tests/dummy/app/helpers/i.js

```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+
+const matcher = /\$\{\s*(.*?)\s*\}/g;
+
+export function interpolate([ str ], hash) {
```

Comment:
this is all looking great @mrosenberg ! Do you still need this helper and the dependency in package.json? Otherwise - I think this looks good to merge!